src/base58.c

```diff
@@ -26,50 +26,50 @@ static const uint32_t MAX_BUFFER_SIZE = 124;
 static const char BASE58ALPHABET[] = "123456789ABCDEFGHJKLMNPQRSTUVWXYZabcdefghijkmnopqrstuvwxyz";
 
 size_t encode_base58(
-        const uint8_t *in, size_t length,
-        uint8_t *out, size_t maxoutlen
+        const uint8_t* inBuffer, size_t inSize,
+        uint8_t* outBuffer, size_t maxOutSize
 )
 {
-	uint8_t tmp[MAX_BUFFER_SIZE];
+	uint8_t tmpBuffer[MAX_BUFFER_SIZE];
 	uint8_t buffer[MAX_BUFFER_SIZE * 2];
 	size_t j;
 	size_t startAt;
 	size_t zeroCount = 0;
 
-	ASSERT(length <= MAX_BUFFER_SIZE);
+	ASSERT(inSize <= MAX_BUFFER_SIZE);
 
-	os_memmove(tmp, in, length);
+	os_memmove(tmpBuffer, inBuffer, inSize);
 
-	while ((zeroCount < length) && (tmp[zeroCount] == 0)) {
+	while ((zeroCount < inSize) && (tmpBuffer[zeroCount] == 0)) {
 		++zeroCount;
 	}
-	j = 2 * length;
+	j = 2 * inSize;
```

Comment:
if `j` isn't used before this point, just move the declaration here. Also with other variables

Reply:
Do you know why is this arbitrary `2 * inSize` ? I mean, would it work if it was `SIZEOF(buffer)` instead?  Also -- do you know why don't we need to clear the rest of the buffer?